### ISSUE TITLE
Table name case sensitivity issue

### ISSUE THREAD

**Issue by jeffbromberger** [SapientGuardian/SapientGuardian.EntityFrameworkCore.MySql#25]
I have a DbSet<Region> and when I look at the code generated for the migration it shows:

```
            migrationBuilder.CreateTable(
                name: "Region",
                columns: table => new
                {
                    Id = table.Column<long>(nullable: false)
                        .Annotation("MySQL:AutoIncrement", true),
                    Name = table.Column<string>(nullable: true)
                },
                constraints: table =>
                {
                    table.PrimaryKey("PK_Region", x => x.Id);
                });
```
However, in the database, the table is getting created as "region" (all lowercase).  I checked the MySQL lower_case_table_names option and it is 0 which indicates that it will store the name however it is specified in the create.  This tells me that either the provider or EF (I don't know who actually creates the table) is converting that Region to region in the SQL for the create, but still requesting Region in later accesses.  On Windows this isn't an issue because the table names are case insensitive due to the file system being case insensitive.  However, when I just deployed to AWS, who apparently runs their MySQL servers on Unix, now my code breaks.  I get an exception saying table "Region" not found even though I can see in workbench that there is a "region" table.

**Comment by SapientGuardian** [comment]
I'm not able to reproduce this. I ran that exact code, and captured the SQL as it was sent across the wire to my DB:

```
CREATE TABLE `Region` (
    `Id` bigint NOT NULL AUTO_INCREMENT,
    `Name` varchar(255),
    PRIMARY KEY (`Id`)
)
```

I also verified that the table in the database was created as 'Region'.

Can you run a packet capture to see if you get the same? That should tell us if the problem is server side or client side.

**Comment by jeffbromberger** [comment]
You're right.  I had tried to inspect the raw SQL earlier but didn't have any luck with wireshark since my db is local.  I was able to figure out how to enable internal tracing in MySQL and then go back at look at it that way and the CREATEs are coming in as Region.  I then did another test where I manually tried to create a table called 'Foo' in workbench and it shows up as 'foo'.

Sorry for the false alarm, but definitely appreciate the quick response and help in getting it narrowed down!
